From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Thu, 12 Aug 2021 20:38:51 +0200
Subject: [PATCH] Rename glib/gio mount function to gio_mount

Somehow glib picks up on Serenity's mount function and gets confused
---
 gio/gio-tool-mount.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/gio/gio-tool-mount.c b/gio/gio-tool-mount.c
index c624268..24723c5 100644
--- a/gio/gio-tool-mount.c
+++ b/gio/gio-tool-mount.c
@@ -330,7 +330,11 @@ new_mount_op (void)
 
 
 static void
+#ifdef __serenity__
+gio_mount (GFile *file)
+#elif
 mount (GFile *file)
+#endif
 {
   GMountOperation *op;
 
@@ -1241,7 +1245,7 @@ handle_mount (int argc, char *argv[], gboolean do_help)
           else if (mount_eject)
             eject (file);
           else
-            mount (file);
+            gio_mount (file);
           g_object_unref (file);
         }
     }
